feat: Expose StopWordFilter from Tantivy (#2577)
paradedb/paradedb: docs/documentation/indexing/token_filters.mdx

    }'
);
```

## Stopwords Language

> <Note>Stopwords Language isnt't supported with ngram tokenizer.</Note>

Reviewer: ```suggestion
<Note>This filter is not supported for the ngram tokenizer.</Note>
```

> `stopwords_language` is a filter that removes common words from the text, which can be applied to any word tokenizer.

Reviewer: ```suggestion
`stopwords_language` removes common "stop words" for a specific language from the original text before tokenization.
```

```sql
CREATE INDEX search_idx ON mock_items
USING bm25 (id, description)
WITH (
    key_field='id',
    text_fields='{
        "description": {"tokenizer": {"type": "default", "stopwords_language": "English"}}
    }'
);
```

<ParamField body="stopwords_language">

>   Available stemmers are `Arabic`, `Danish`, `Dutch`, `English`, `Finnish`,

Reviewer: ```suggestion
  Available languages are `Arabic`, `Danish`, `Dutch`, `English`, `Finnish`,
```

Reviewer: I noticed the "stemmers" typo - can you verify these languages are correct?

Author: Great catch!! Updated and also added testing in https://github.com/paradedb/paradedb/pull/2577/commits/27fab5c3ee4f90c4d8db1c4f132db7b13a29b746

  `French`, `German`, `Greek`, `Hungarian`, `Italian`, `Norwegian`,
  `Portuguese`, `Romanian`, `Russian`, `Spanish`, `Swedish`, `Tamil`, and
  `Turkish`.
</ParamField>

## Custom Stopwords

> <Note>Custom Stopwords Language isnt't supported with ngram tokenizer.</Note>

Reviewer: ```suggestion
<Note>This filter is not supported for the ngram tokenizer.</Note>
```

> `stopwords` is a filter that removes custom words from the text. The `stopwords` filter can be applied to any tokenizer.

Reviewer: ```suggestion
`stopwords` removes custom words from the original text before tokenization.
```
